Lib/test/test_mimetypes.py

```diff
@@ -285,50 +287,54 @@ def test__all__(self):
 
 class MimetypesCliTestCase(unittest.TestCase):
 
-    def mimetypes_cmd(self, *args, **kwargs):
-        support.patch(self, sys, "argv", [sys.executable, *args])
-        with support.captured_stdout() as output:
-            mimetypes._main()
-            return output.getvalue().strip()
+    @classmethod
```

Comment:
nit: this method is never used as `MimetypesCliTestCase.mimetypes_cmd`, no need to mark it as `classmethod`.